apps/src/gamelab/GameLabSprite.js

```diff
@@ -14,6 +14,16 @@ module.exports.createSprite = function (x, y, width, height) {
   var s = new this.Sprite(x, y, width, height);
   var p5Inst = this;
 
+  s.setAnimation = function (animationName) {
+    var animation = p5Inst.projectAnimations[animationName];
```

Comment:
I'm just putting the preloaded animations onto the p5 instance - I wasn't sure if there's a better place to put them where GameLabSprite would be able to access them.  Any ideas?